src/client/runner/automation/playback/key-press-simulator.js

```diff
@@ -580,20 +593,19 @@ var supportedShortcutHandlers = (function () {
                 else {
                     //the form is also submitted on enter press if there is only one input of the following types on it
                     //  and this input is focused (http://www.w3.org/TR/html5/forms.html#implicit-submission)
-                    var textInputTypeRegExp = browserUtils.isWebKit ? /^(text|search|url|number|tel|password|number|email)$/i :
-                                              /^(text|search|url|number|tel|password|number|email|date|time)$/i;
+                    var textInputTagNameRegExp = getTextInputTagNameRegExp();
```

Comment:
standard calls such fields *fields that blocks implicit submission*
so, maybe `blocksImplicitSubmission(el)`